include/wayfire/config/compound-option.hpp

```diff
@@ -30,6 +30,12 @@ class compound_option_entry_base_t
         return prefix;
     }
 
+    /** @return The name of the tuple entry. */
+    virtual std::string get_name() const
```

Comment:
nitpick: this doesn't have to be virtual, because it is not overridden in the subclasses.

Reply:
@ammen99 I just assumed adding virtual was future-proofing. We have a `virtual` for `get_prefix()` as well, and it's not overridden in the subclasses, either.

Reply:
Removed `virtual` from both.